indico/modules/events/tracks/models/abstract_reviewers.py

```diff
@@ -23,20 +23,31 @@
     'track_abstract_reviewers',
     db.metadata,
     db.Column(
+        'id',
+        db.Integer,
+        primary_key=True,
+        autoincrement=True,
```

Comment:
that's implied for "simple" single-column PKs; i was only setting it to false because the behavior for composite PKs is broken/unexpected in sqlalchemy